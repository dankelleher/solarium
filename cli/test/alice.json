[14,108,199,154,11,192,160,109,48,180,94,79,238,169,155,67,12,198,222,216,158,245,35,47,195,158,131,52,83,208,223,154,161,62,198,166,99,106,237,235,191,2,50,171,73,194,84,53,92,145,159,239,169,112,125,86,203,166,214,207,33,181,255,179]