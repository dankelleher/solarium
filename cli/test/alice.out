Generating a new keypair

Wrote new keypair to alice.json
====================================================================
pubkey: BrSACyPFCpXMug2LjWVGSxnHeiFSpZCrFQvFYyMn1f3Q
====================================================================
Save this seed phrase and your BIP39 passphrase to recover your new keypair:
mind save maple pig proof light envelope ability river lazy pole toe
====================================================================
